src/apiCalls.js

```diff
@@ -0,0 +1,27 @@
+import { displayMessage, displayScores } from './display';
+
+const gameId = '3L4QvyqAZFTD6eSge8xk';
+
+export async function submitScore(name, score) {
```

Comment:
- [ ] Ensure the usage of arrow functions instead of the function keyword.

Reply:
Done